Fakebook.Posts/Fakebook.Posts.RestApi/DTOs/EditPostDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Fakebook.Posts.Domain.Constants;
+
+namespace Fakebook.Posts.RestApi.DTOs
+{
+    public class EditPostDTO
+    {
+        [Required]
+        public int PostId { get; set; }
```

Comment:
if we don't want to support updates to PostId, we should just leave it out of here entirely. based on the URL we know what post is to be updated

Reply:
I see that now didn't notice post Id gets passed in as a parameter, I removed it from that EditPostDTO.  That DTO doesn't have implementation as of now so I'm going to leave it at that and keep that in mind for when we modify the Put method in that controller to take the DTO